styles/_base.css

```
@@ -3,19 +3,64 @@
 }
 
 body {
-  font-family: sans-serif;
-  font-size: 112.5%;
-  line-height: 1.5;
-  margin: 0;
-}
-
-h1,
-h2,
-h3 {
-  line-height: 1.3;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 14px; /* 112.5% */
```

Please set the font-size to 112.5%. `14px` will overwrite browser settings which we don't want.
```suggestion
  font-size:  112.5%;
```